ocean/coupling_files_E3SM/create_E3SM_MPAS_coupling_files.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+"""
+This script creates coupling files needed to run an MPAS-Ocean and
+MPAS-seaice within E3SM.
+"""
+import os
+import shutil
+import subprocess
+import ConfigParser
+import numpy as np
+
+
+def main():
+
+    config = ConfigParser.ConfigParser()
```

Comment:
To support python 3, this would become:
```suggestion
    config = ConfigParser()
```